### PR TITLE
addpkg: koko

### PR DESCRIPTION
**Pull request by Integral-Tech**

### CODE REVIEW

File: koko/riscv64.patch [hunk 1]
```diff
@@ -0,0 +1,24 @@
+--- PKGBUILD
++++ PKGBUILD
+@@ -5,7 +5,7 @@ pkgver=23.04.0
+ pkgrel=1
+ pkgdesc='Image gallery application'
+ url='https://apps.kde.org/koko/'
+-arch=(x86_64 riscv64)
++arch=(x86_64)
```
Comment (Xeonacid) [inline koko/riscv64.patch:8]: Is this committed by mistake? (We don't need the reversing `arch` change also here)

**Review by Xeonacid — CHANGES_REQUESTED** [review]
Please report the checksum mismatch to Arch.

File: koko/riscv64.patch (hunk 1 above)
Comment (Integral-Tech) [reply koko/riscv64.patch:8]: Yes, it is committed by mistake :(

**Comment by Integral-Tech** [comment]
> Please report the checksum mismatch to Arch.

Therefore, should I close this pull request?

**Comment by Xeonacid** [comment]
Sorry for my misunderstanding words. We could merge it here for a quick fix, but should **also** report it to Arch. :)

**Comment by Integral-Tech** [comment]
> Sorry for my misunderstanding words. We could merge it here for a quick fix, but should **also** report it to Arch. :)

OK :)

**Review by Xeonacid — CHANGES_REQUESTED** [review]
Check still fails:
```text
patch: **** malformed patch at line 10: @@ -16,9 +16,9 @@ source=(https://download.kde.org/stable/release-service/$pkgver/src/$pkgname-$pk
```

**Comment by r-value** [comment]
Please re-generate the patch instead of modifying it manually.

**Review by Xeonacid — APPROVED** [review]

**Comment by felixonmars** [comment]
Upstream report at https://bugs.archlinux.org/task/78432

**Pull request merged**